utoipa-axum/src/router.rs

```diff
@@ -10,14 +10,6 @@ use axum::Router;
 use tower_layer::Layer;
 use tower_service::Service;
 
-#[inline]
-fn colonized_params<S: AsRef<str>>(path: S) -> String
-where
-    String: From<S>,
-{
-    String::from(path).replace('}', "").replace('{', ":")
-}
-
 #[inline]
 fn path_template<S: AsRef<str>>(path: S) -> String {
```

Comment:
We can most likely remove this `path_template` fn as well since it is used to counter the colonized syntax and translate colonized paths to path template syntax, but since axum 0.8 comes with path template syntax this because redundant :thinking: